identity/client/src/pages/tenants/modals/AddModal.tsx

```diff
@@ -61,8 +67,12 @@ const AddTenantModal: FC<UseModalProps> = ({ open, onClose, onSuccess }) => {
         <TextField
           label={t("tenantId")}
           placeholder={t("tenantIdPlaceholder")}
-          onChange={setTenantId}
+          onChange={(value) => {
+            validateTenantId(value);
+            setTenantId(value);
```

Comment:
For accessibility reasons it's better to run the validation on submit
Could we change that?